src/test/scala/viper/gobra/GobraPackageTests.scala

```diff
@@ -22,58 +24,87 @@ import scala.concurrent.duration.Duration
 class GobraPackageTests extends GobraTests {
   override val testDirectories: Seq[String] = Vector("same_package")
 
+  override def buildTestInput(file: Path, prefix: String): DefaultAnnotatedTestInput = {
```

Comment:
@Felalolf I've found out that our testing frameworks supports test cases that span multiple files. The advantage of using that feature is that that test case will only be executed once (instead of once per file that belongs to it). The previous implementation of GobraPackageTests suffered from this.
`buildTestInput` is very similar to package resolution as it retrieves the list of gobra files in the current directory that share the same package name and creates a test input with that list of files